gluten-ut/spark32/src/test/scala/org/apache/spark/sql/execution/joins/GlutenBroadcastJoinSuite.scala

```diff
@@ -19,22 +19,40 @@ package org.apache.spark.sql.execution.joins
 
 import io.glutenproject.backendsapi.BackendsApiManager
 import io.glutenproject.GlutenConfig
+import io.glutenproject.execution.{GlutenBroadcastHashJoinExecTransformer, WholeStageTransformerExec}
```

Comment:
‘GlutenBroadcastHashJoinExecTransformer ’ is only for velox backend, using `BroadcastHashJoinExecTransformer` please

Reply:
thank you @zzcclp , fixed, please help review. thank you again.

Reply:
@rui-mo I remember we talked to rename the transformer and remove Gluten prefix. Right?

Reply:
Yes, that's right. We'd better remove Gluten prefix because it is apparent.

Reply:
Can you submit a PR to remove it? @zzcclp have you removed all on CH side?

Reply:
Sure. Will do.

Reply:
On CH side, the prefix is CH, not the Gluten.

Reply:
Since we won't run CH and Velox at the same time, so we may use the same name without prefix.